components/cards/Work.jsx

```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
+import { H3 } from "../ui/Headings";
+import { P } from "../ui/Headings";
+import { DefaultButton } from "../ui/Button";
+import projects from "../../data/projects";
```

Comment:
data folder is in the root path, so it can be accessed directly using one slash `/`.

Reply:
Applied